src/stylesheets/_variables.scss

```diff
@@ -0,0 +1,82 @@
+// Colors
+$grays: (
```

Comment:
@carryall Should we use `$brand-gray` instead of `$grays`?
Because it seems plural word.

Reply:
@Lahphim I use `$grays` because I intend to override the Bootstrap variable `$grays` 🤔  
do you think it makes sense to keep this name or should we declare `$brand-gray` then assign the `$grays` with its value?
![Screen Shot 2563-11-18 at 15 33 55](https://user-images.githubusercontent.com/1772999/99505587-96c04580-29b3-11eb-9d54-4f135525a8d0.png)

Reply:
@carryall I just think about something like this, but I never try on this.
```scss
$brand-gray: (
  ...
);
$grays: map-merge((), $brand-gray)
```
the benefit of this is about using `$brand-gray` in our system also 🤣

Reply:
✅  updated in [16cb735](https://github.com/carryall/survey-challenge/pull/2/commits/16cb735c3e01d450a83b200ace3880f02e2f1634)